pkg/cmd/audit/audit_filter.go

```diff
@@ -177,6 +188,29 @@ func (f *FilterByResources) Matches(event *auditv1.Event) bool {
 	return false
 }
 
+func getFieldManagerQualifiedUsername(event *auditv1.Event) string {
+	username := event.User.Username
+	if fieldManager := QueryParams(event.RequestURI).Get("fieldManager"); len(fieldManager) > 0 {
+		username = fmt.Sprintf("%s[%s]", username, fieldManager)
+	}
+	return username
+}
+
+func QueryParams(uri string) url.Values {
+	// some request URL has query parameters like: /apis/image.openshift.io/v1/images?limit=500&resourceVersion=0
+	// we are not interested in the query parameters.
+	parts := strings.Split(uri, "?")
+	if len(parts) < 2 {
+		return nil
```

Comment:
won't this blow up?

Reply:
no.  `url.Values` is a map and retrieval from a nil map is valid in golang.